app/routes/index.js

```diff
@@ -65,12 +65,26 @@ const checkBoxes = (data, question) => {
   }
 }
 
+const inputBoxes = (question) => {
+  return {
+    id: question.id,
+    name: question.name,
+    classes: 'govuk-input--width-10',
+    prefix: question.prefix,
+    suffix: question.suffix,
+    label: question.label,
+    hint: question.hint
+  }
+}
+
 const getOptions = (data, question) => {
   switch (question.type) {
     case 'single-answer':
       return radioButtons(data, question)
     case 'multi-answer':
       return checkBoxes(data, question)
+    case 'input':
+      return inputBoxes(question)
```

Comment:
i think we have to pass data as well so we display the user data if the user is revisiting